lib/ui/platform_dispatcher.dart

```diff
@@ -1155,7 +1155,7 @@ class PlatformDispatcher {
       onSemanticsAction,
       _onSemanticsActionZone,
       nodeId,
-      SemanticsAction.values[action]!,
+      SemanticsAction.deserialize(action)!,
```

Comment:
Since the getter is called `value` maybe we should just call this `SemanticsAction.fromValue`.

Reply:
Alternatively, we could keep the old name `index` for the property name and call this `SemanticsAction.fromIndex`.